lib/impl/editing.dart

```diff
@@ -91,6 +88,19 @@ bool _handleEnterKey(TextEditor editor, int row, int col) {
     return true;
   }
 
+  if (trimmedText.startsWith('//')) {
```

Comment:
If the line started with '///', I believe it would have been handled by the if preceding this one.